test/testdata/lsp/completion/constr.rb

```diff
@@ -4,12 +4,7 @@
 
 xs = T.let([], T::Array[Integer])
 
-# This completion result is a bit non-sensical at the moment (it doesn't makes
-# sense to return any results there) but at least it doesn't crash Sorbet like
-# it used to. If you made a change that improves this test, feel free to delete
-# this comment.
-
 xs.each.map do |x|
   x
 end
-#  ^ completion: each, ...
+#  ^ completion: (nothing)
```

Comment:
Why doesn't this return any completion results? Wouldn't the following be an example of where you'd want to allow completion after the `end`?
```ruby
irb(main):001:0> xs = [1,2,3]
=> [1, 2, 3]
irb(main):002:1* xs.map do |a|
irb(main):003:1*   a+1
irb(main):004:1* end.each do |a|
irb(main):005:1*   puts a
irb(main):006:0> end
2
3
4
=> [2, 3, 4]
```

Reply:
You haven't typed the `.` yet. There's nothing to complete until you type the `.`

Reply:
Ah, that makes sense. Thanks!